### ISSUE TITLE
Mismatch of "package.el" and "LAYERS.org"

### ISSUE THREAD

**Issue by AlohaWorld** [syl20bnr/spacemacs#6386]
#### Description
When using "configuration-layer/create-layer" to create a new layer, the "package.el" uses a keyword "defconst", while "LAYERS.org" uses "setq"

#### Reproduction guide
- Start Emacd:/home/cyd/.spacemacs.private/my-cygwin/

#### System Info
- OS: windows-nt
- Emacs: 24.5.1
- Spacemacs: 0.105.21
- Spacemacs branch: cyd (rev. 0283f64)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-return-key-behavior 'complete auto-completion-tab-key-behavior 'cycle auto-completion-enable-snippets-in-popup t auto-completion-complete-with-key-sequence nil auto-completion-complete-with-key-sequence-delay 0.1 auto-completion-private-snippets-directory
                  (concat
                   (car dotspacemacs-configuration-layer-path)
                   "snippets/")
                  auto-completion-enable-sort-by-usage t)
 better-defaults emacs-lisp git org
 (shell :variables shell-default-height 30 shell-default-position 'bottom)
 spell-checking syntax-checking themes-megapack chinese my-cygwin)
```

**Comment by d12frosted** [comment]
Hey,

Documentation was [already updated](https://github.com/syl20bnr/spacemacs/blob/develop/doc/LAYERS.org#packagesel) in `develop` branch. 

I am not sure why it's not on `master` yet, so thanks for bringing it up.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
